CMakeLists.txt

```diff
@@ -148,11 +108,64 @@ set(CLP_FFI_JS_SRC_ZSTD
     src/submodules/zstd/lib/decompress/zstd_decompress.c
 )
 
-target_sources(
-    ${CLP_FFI_JS_BIN_NAME}
-    PRIVATE
+# List of supported environments.
+set(CLP_FFI_JS_ENVIRONMENTS
```

Comment:
* How about `CLP_FFI_JS_SUPPORTED_ENVIRONMENTS`? Then we can drop the comment above.
* Can we make this a cache variable so that we can set it from the Taskfile? That way, we don't have to keep the environments here in sync with the Taskfile.